python/ray/serve/_private/client.py

```diff
@@ -135,7 +135,15 @@ def _wait_for_deployments_shutdown(self, timeout_s: int = 60):
         """
         start = time.time()
         while time.time() - start < timeout_s:
-            deployment_statuses = self.get_serve_status().deployment_statuses
+            statuses_bytes = ray.get(
```

Comment:
Could we pull this protobuf parsing into a new helper function that's similar to `get_serve_status()`? We could reuse it outside this function.